### DISCUSSION TITLE
History "suggestions"

### DISCUSSION THREAD

**Discussion by hastiin-manuelito** [xonsh/xonsh#4816] [Q&A] [answered]
Is there a way to partially "complete" the history entries that are suggested (as opposed to tab-completing) as you type a command in the prompt?

**Comment by jnoortheen** [comment, marked as answer]
Alt+f will complete the next word instead of the whole line

**Reply by amitkot** [reply]
Is this a Linux specific behavior? In macOS when I press Alt+f I get instead `ƒ` printed to the terminal.

**Reply by jnoortheen** [reply]
you have to configure your terminal to pass Alt key unmodified on macos ITerm
